carbon-design-system/carbon
# fix: 15524-overflow-menu-focus-bug

Closes #15524  & #15366

This should fix the focus problem. Decoupled escape focus with other focus logic. 

#### Testing
- Open [this storybook link](https://deploy-preview-15833--v11-carbon-react.netlify.app/?path=/docs/components-overflowmenu--overview)
- Open the first overflow menu - focus should remain within the current overflow menu 
- Open the overflow menu further down in the documentation - focus should remain within the second overflow menu 
- Overflow menus should close and maintain focus on escape
